### PR TITLE
RM-265655 Release dpx 0.1.1

### PR DESCRIPTION
**Pull request by rmconsole-readonly-wk**
Pull Requests included in release:
* Patch changes:
	* [FEDX-965: Readme updates](https://github.com/Workiva/dpx/pull/10)
	* [FEDX-967: Lengthen package description](https://github.com/Workiva/dpx/pull/13)
	* [FEDX-969: Add GHA workflow to automate publishing to pub.dev](https://github.com/Workiva/dpx/pull/14)

Requested by: @evanweible-wf

Diff Between Last Tag and Proposed Release: https://github.com/Workiva/dpx/compare/0.1.0...Workiva:release_dpx_0.1.1
Diff Between Last Tag and New Tag: https://github.com/Workiva/dpx/compare/0.1.0...0.1.1

The logs for the request that created this PR can be found [here](https://w-rmconsole.appspot.com/release/release_event/5176949306163200/logs/)
This pull request can be recreated by clicking [here](https://w-rmconsole.appspot.com/api/v2/rosie/reTriggerReleaseEvents/5176949306163200/?repo_name=Workiva%2Fdpx&pull_number=22)

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: pubspec.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: dpx
-version: 0.1.0
+version: 0.1.1
```
Comment (matthewnitschke-wk) [inline pubspec.yaml:2]: #nit, I noticed that [`version.dart`](https://github.com/Workiva/dpx/blob/master/lib/src/version.dart) isn't getting updated

meaning `dpx --version` is outputting an incorrect version

Not blocking, but is a bit confusing especially given the issue template asking to run `dpx --version`

**Review by matthewnitschke-wk — APPROVED** [review]

File: pubspec.yaml (hunk 1 above)
Comment (evanweible-wf) [reply pubspec.yaml:2]: I totally forgot about that :) I'll fix that up in a separate PR.

**Comment by evanweible-wf** [comment]
QA +1 @Workiva/release-management-p

**Comment by evanweible-wf** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**